### ISSUE TITLE
data clustering and split 

### ISSUE THREAD

**Issue by elgeekim** [tttianhao/CLEAN#42]
1. I clustered 'split100.csv' with MMseqs using a 0.5 identity condition, and there were 32,283 clusters, 
    whereas 'split50.csv' resulted in 29,942, showing a difference. 
    (mmseqs cluster --min-seq-id 0.5)
    Was there any additional data removal after the clustering? 
    If the clustering conditions were different, please let me know how it was done.

2. Can you provide specific details on how you split the data for cross-validation? 
    If you did a random split, could you inform me about the random state used?

3. What is the reason for not selecting models using a separate validation set or 
    utilizing models from cross-validation for benchmark testing?

**Comment by canallee** [comment]
Hi I have just uploaded the 5-fold CVs for all the splits. Please refer to the README. These CVs were used for Fig. 2, Fig. S1, Fig. S2.

**Comment by tttianhao** [comment]
The data split was random, and after splitting, all EC numbers that only exist in the validation set but not in the training set were removed.
